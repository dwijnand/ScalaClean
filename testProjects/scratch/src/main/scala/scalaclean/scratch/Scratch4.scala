package scalaclean.scratch


class Scratch4 {

  var myVar: Int  = 1
}
