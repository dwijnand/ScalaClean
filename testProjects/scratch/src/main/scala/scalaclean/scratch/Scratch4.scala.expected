package scalaclean.scratch


/* *** SCALA CLEAN can't detect usage */class Scratch4 {

  /* *** SCALA CLEAN can't detect usage */var myVar: Int  = 1
}
